android/src/main/java/com/swmansion/rnscreens/ScreenWindowTraits.kt

```diff
@@ -197,7 +197,6 @@ object ScreenWindowTraits {
         val screenForNavBarHidden = findScreenForTrait(screen, WindowTraits.NAVIGATION_BAR_HIDDEN)
         val hidden = screenForNavBarHidden?.isNavigationBarHidden ?: false
 
-        WindowCompat.setDecorFitsSystemWindows(window, hidden)
```

Comment:
I've discussed that change with @kkafar and we think that after hiding navigation bar / status bar the effect looks better without calling this method - the view doesn't jump without setting `fitsSystemWindows` to DecorView.

Reply:
Resolved in separate PR: https://github.com/software-mansion/react-native-screens/pull/1988